src/utils/lib.ml

```diff
@@ -58,6 +65,14 @@ let mk_dir dir =
    to a negtive number *)
 let safe_hash_interleave h m i = abs(i + (m * h) mod max_int)
 
+let max_of: int -> int -> int = fun a b ->
+  if a > b then a else b
+(** returns the maximum of the integer *)
+
+let abs_diff: int -> int -> int = fun a b ->
+  if a > b then a - b else b - a 
+(** Returns the absolute difference of the two integers *)
+  
```

Comment:
I didn’t find any place where you are using these functions, but please consider using the standard functions `Stdlib.max` and `Stdlib.abs`.

Reply:
This is removed now.